internals/notifier.py

```diff
@@ -722,6 +722,44 @@ def make_extension_request_email(self, extension_stage, ot_stage):
       'html': email_body,
     }
 
+
+class OriginTrialExtendedHandler(basehandlers.FlaskHandler):
+  """Notify about an origin trial extension being completed."""
+
+  IS_INTERNAL_HANDLER = True
+
+  def process_post_data(self, **kwargs):
+    extension_stage = self.get_param('stage')
+    ot_stage = self.get_param('ot_stage')
+    logging.info('Starting to notify about successful origin triale extension.')
+    send_emails([self.build_email(self, extension_stage, ot_stage)])
+
+  def build_email(self, extension_stage, ot_stage):
+    email_body = f"""
+<p>
+  A recent extension request was finalized and processed.
+  <br>
+  Requested by: {extension_stage["ot_owner_email"]}
```

Comment:
I think that you should use a Jinja2 template for this instead of a python f-string.  The reason is that Jinja2 will automatically do context-sensitive escaping.  We expect the emails to be safe but something might slip through. And, the display name is user controlled with no restrictions.  So, it would be better for all html-generating code to have a layer of defense against malicious input.

Reply:
You're definitely right. I made the temporary emails used for trial extension/creation notifications and followed that format, which was always meant to be removed. I've updated the implementation to use a template.